### PR TITLE
Fixes #8908: Document separate database configuration in Rudder

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8908

### CODE REVIEW

**Comment by amousset** [comment]
Do not merge for now, still having some CSS issues.

File: 4_advanced_usage/75_separate_database.txt [hunk 1]
```diff
@@ -6,6 +6,85 @@ The setup is done in two places: on the database server and on the Rudder root s
 
 It also allows moving an existing database to another server.
 
+[TIP]
+
+.Use different user and database name
```
Comment (jooooooon) [inline 4_advanced_usage/75_separate_database.txt]: NameS

File: 4_advanced_usage/75_separate_database.txt [hunk 2]
```diff
@@ -6,6 +6,85 @@ The setup is done in two places: on the database server and on the Rudder root s
...
+
+====
+
+It can be useful, for example if you want to share you databse server between several Rudder root server (see note below),
```
Comment (jooooooon) [inline 4_advanced_usage/75_separate_database.txt]: Database

Comment (jooooooon) [inline 4_advanced_usage/75_separate_database.txt]: ServerS

File: 4_advanced_usage/75_separate_database.txt [hunk 3]
```diff
@@ -6,6 +6,85 @@ The setup is done in two places: on the database server and on the Rudder root s
...
+
+.Use the same database server for several Rudder root servers
+
+====
+
+It is possible to share the same database server between several Rudder instances,
+by following the preceding tip to use a different database than the default one.
+However, there are some important points to know:
+
+* This database server can only be used as database and not for reports insertion
```
Comment (jooooooon) [inline 4_advanced_usage/75_separate_database.txt]: I don't understand this point. Could you clarify please?

**Comment by amousset** [comment]
Commit modified

File: 4_advanced_usage/75_separate_database.txt [hunk 4]
```diff
@@ -6,6 +6,84 @@ The setup is done in two places: on the database server and on the Rudder root s
...
+* Then apply the script:
+
+----
+
+su - postgres -c "psql -q -U alternate_user_name -h localhost -d alternate_base_name \
+     -f /opt/rudder/etc/postgresql/reportsSchema-alternate.sql"
+
+----
+
+* Follow the normal instructions, with two differences :
```
Comment (jooooooon) [inline 4_advanced_usage/75_separate_database.txt]: What normal instructions? Can you clarify, as in "above" or with a link?

BTW, "normal" is a word best avoided. I propose "standard" instead.

Comment (jooooooon) [inline 4_advanced_usage/75_separate_database.txt]: Also, no space before the colon in English ("differences:")

**Comment by amousset** [comment]
Commit modified

**Pull request merged**